README.rst

```diff
@@ -73,7 +73,7 @@ Flair
 .. image:: https://img.shields.io/pypi/v/eventlet
     :target: https://pypi.org/project/eventlet/
 
-.. image:: https://img.shields.io/github/workflow/status/eventlet/eventlet/test/master
+.. image:: https://img.shields.io/github/actions/workflow/status/eventlet/eventlet/test.yaml?branch=master
```

Comment:
That's looks good, however, what do you think about relying directly on github to display github badges?

Example:
```
[![test](https://github.com/eventlet/eventlet/actions/workflows/test.yaml/badge.svg)](https://github.com/eventlet/eventlet/actions/workflows/test.yaml)
```

Reply:
Looks good to merge.